azurelinuxagent/common/osutil/default.py

```diff
@@ -99,6 +102,11 @@ def _get_firewall_delete_conntrack_accept_command(wait, destination):
                       "--ctstate", "INVALID,NEW", "-j", "ACCEPT"])
 
 
+def _get_firewall_delete_accept_dns_tcp_command(wait, destination):
```

Comment:
@larohra - do you mean calling get_iptables_accept_dns_tcp_request_command from here? I think the current implementation is clearer.

Reply:
yes (with better name for the function ofcourse), but yeah that's the idea. From the implementation, it looks like the only thing different is the command we pass to it (`-A` vs `-D`). If that is indeed the case, I would rather have one single function for it rather than copies of the same thing to avoid code duplication primarily

Reply:
OK, we can try to reuse code, but the previous abstraction was an overkill and made the code really hard to verify.  I am not concerned about having some duplication for just 4 commands. The resulting code should be fairly simple and easy to read. I think a few array constants and array concatenation would do the trick, but @dhivyaganesan can decide that.

Reply:
I totally agree with you, array constants and concatenations would solve my comments too :)

Reply:
I haven't moved the delete rule from default to networkutil yet. I will do it when I modify all the rules and move everything to networkutil. This required a remane in many test files and I feel its easier to do in one sitting for all rules after the logic change is implemented